app/src/db/schema.ts

```diff
@@ -1,11 +1,58 @@
-import { integer, pgTable, serial, text } from 'drizzle-orm/pg-core';
+import { boolean, integer, pgTable, serial, text, timestamp } from 'drizzle-orm/pg-core';
 
-export const usersTestTable = pgTable('users_test_table', {
+export const orders = pgTable('orders', {
   id: serial('id').primaryKey(),
-  name: text('name').notNull(),
-  age: integer('age').notNull(),
-  email: text('email').notNull().unique(),
+  userId: text('user_id').notNull(),
+  orderDate: timestamp('order_date').notNull(),
+  shipmentDate: timestamp('shipment_date'),
+  estimatedDeliveryDate: timestamp('estimated_delivery_date'),
+  actualDeliveryDate: timestamp('actual_delivery_date'),
+  lastCommunicationAt: timestamp('last_communication_at'),
+  relatedCommunicationIds: text('related_communication_ids').array(),
+  trackingUrls: text('tracking_urls').array(),
+  trackingNumbers: text('tracking_numbers').array(),
+  createdAt: timestamp('created_at').defaultNow().notNull(),
+  updatedAt: timestamp('updated_at').defaultNow().notNull(),
 });
 
-export type InsertUser = typeof usersTestTable.$inferInsert;
-export type SelectUser = typeof usersTestTable.$inferSelect;
+export type InsertOrder = typeof orders.$inferInsert;
+export type SelectOrder = typeof orders.$inferSelect;
+
+export const communications = pgTable('communications', {
+  id: serial('id').primaryKey(),
+  userId: text('user_id').notNull(),
+  communicationType: text('communication_type').notNull(), // 'email' for now, can think about texts later'
+  subject: text('subject'),
+  content: text('content').notNull(),
```

Comment:
For privacy reasons, you might not want to store the message content in your database.